taxonomy-editor-frontend/src/pages/search/SearchResults.jsx

```diff
@@ -18,9 +17,12 @@ import DialogContent from '@mui/material/DialogContent';
 import DialogTitle from '@mui/material/DialogTitle';
 import Select from '@mui/material/Select';
 import ISO6391 from 'iso-639-1';
+import { createBaseURL } from "../editentry/createURL";
 
-const SearchResults = ({query}) => {
-    const { data: nodeIds, isPending, isError, isSuccess, errorMessage } = useFetch(`${API_URL}search?query=${encodeURI(query)}`);
+const SearchResults = ({query, taxonomyName, branchName}) => {
+    const url = createBaseURL(taxonomyName, branchName);
```

Comment:
nit: `baseUrl`? think at least it gives a hint that is comming from the `createBaseURL` function, and `url` can be a bit more cryptic